RELEASE-NOTES.md

```diff
@@ -27,6 +27,9 @@
   * `snow snowpark deploy` uploads all artifacts created during build step. Dependencies zip is upload once to
      every Snowpark stage specified in project definition.
   * The changes are compatible with V1 projects definition though the result state (file layout) is different.
+  * `snow snowpark package` commands no longer fallback to Anaconda Channel metadata when fetching available packages
+    info fails.
```

Comment:
Is this related to previous line?

Reply:
Yes, "fetching available packages info fails"

Reply:
Ok. Could you move to one line?

Reply:
Done